### PR TITLE
matplotlib version fix

### PR DESCRIPTION
**Pull request by yuta-imai**
[Issue]
In Jupyter Notetook, importing cv2 and matplotlib failes.

- For cv2: kernels dies.
- For matplotlib: it says `RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd`

[Cause]

We install the latest matplotlib when deploying this packaget to S+ Camera. It looks like matplotlib has been updated and now it's not compatible to S+ Camera platform's numpy version(0xd).

[Solution]

We fix matplotlib version to 3.3.2.

### CODE REVIEW

File: CameraApp0/PreSetup [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 pip install notebook
```
Comment (ks6088ts) [inline CameraApp0/PreSetup]: The version of `notebook` is not fixed. This has a potential runtime error in the long run.
Freezing versions would resolve this kind of errors.
For example, you can add `requirements.txt` and run `pip install -r requirements.txt` in PreSetup script.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: test
+
+on: [push]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
```
Comment (yuta-imai) [inline .github/workflows/test.yml:9]: Not sure if we can go with ubuntu, is there any options for raspbian?

File: CameraApp0/PreSetup (hunk 1 above)
Comment (yuta-imai) [reply CameraApp0/PreSetup]: Thanks! Update as you recommended.

**Comment by yuta-imai** [comment]
@ks6088ts Can you review the updated code?

File: .github/workflows/test.yml (hunk 2 above)
Comment (ks6088ts) [reply .github/workflows/test.yml:9]: currently not supported yet.
ref. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

**Review by ks6088ts — APPROVED** [review]

**Comment by ks6088ts** [comment]
> @ks6088ts Can you review the updated code?

Approved.
Thanks for the fix :)

**Pull request merged**